colibre/description.html

```diff
@@ -159,29 +187,60 @@ <h4>AGN</h4>
     </tr>
     <tr>
         <td>Use Krumholtz Vorticity</td>
-        <td>{{ data.metadata.parameters.get("COLIBREAGN:use_krumholz_vorticity", 0) | int }}</td>
+        <td>{{ data.metadata.parameters.get("COLIBREAGN:with_krumholz_vorticity", 0) | int }}</td>
     </tr>
     <tr>
         <td>Use Angular Momentum Limiter</td>
         <td>{{ data.metadata.parameters.get("COLIBREAGN:with_angmom_limiter", 0) | int }}</td>
     </tr>
     <tr>
-        <td>Reposition Velocity</td>
-        <td>{{ data.metadata.parameters.get("COLIBREAGN:reposition_coefficient_upsilon", 0.0) | float | format_number }}</td>
+        <td>Normalisation of the viscous time-scale</td>
```

Comment:
Can we keep this as `viscous_alpha` please? That's more clear to me.

Reply:
Sure